contents/blog/product-health-metrics.md

```diff
@@ -0,0 +1,127 @@
+category: Product growth
+tags:
+ - Guides
+ - Product metrics
+---
+
+Like the human body, a product is a complicated system you want to keep healthy. Measuring product health with metrics helps this happen. They help you understand what parts of your product are working as expected and what parts need interventions to make them healthy.
+
+This post goes over:
+
+1. what makes a good health metric
+2. common product health metrics
+3. how to choose the right product health metrics
+
+## What is a product health metric?
+
+If you think of a product like a human body, then product health metrics are like the vital signs. They are the body temperature, blood pressure, pulse rate, and respiration rate of a product. They are a way to constantly judge how your product is doing. They show how changes, updates, and the environment are impacting it.
+
+Product health metrics are usually steady. Radical changes are a bad sign. Having your temperature way above average means you are sick and need medicine. If a product health metric sees a radical change, it needs intervention.
+
+Companies aim to improve product health metrics steadily over the long term. Like how exercise improves your pulse and respiration rates, adding enhancements and fixing bugs improve your product health metrics.
+
+Because they are constantly measured and steadily improved, product health metrics generally update in real-time and changes are measured over a length of time. For example, a product health metric might be down for the week, which triggers a minor adjustment in focus for next week. It can at the same time be up for the month, which is an encouraging sign. 
+
+Another similarity to your body’s health is that mature products generally care more about them. Early-stage companies are searching for product fit. This often requires product changes or pivots that lead to radical changes in product health metrics. It doesn’t mean the product is "sick," it means they haven’t figured out what "healthy" means yet.
+
+## What makes a good product health metric?
+
+A good health metric is one you monitor regularly and makes a difference if it changes significantly. It must be a metric that matters, and is likely to change if something goes wrong. It is a summary of a product, so it must be abstract enough to remove unnecessary details, but detailed enough to provide actionable information.
+
+The key to actionable information is that you must act on it. If a change in your product health metrics doesn’t prompt a change in your development or strategy, the product health metrics aren’t good. A doctor is useless if you never listen to them. 
+
+For example, if you see a drop in session duration, it might mean your product isn’t engaging enough and you must focus on improving engagement. This should lead you to figure out ways to improve engagement and prioritize that work over other work. If the drop in session duration doesn’t inspire change, it isn’t a good health metric.
```

Comment:
This kind of feels like going over the same ground as the previous section? Perhaps you can just integrate some of these points there?

Reply:
rewrote as one